package.json

```diff
@@ -19,7 +19,7 @@
     "verify:lint": "pnpm lint",
     "verify:types": "tsc --build tsconfig.json",
     "format": "pnpm lint:fix && pnpm style:fix",
-    "precodegen": "pnpm run --recursive --stream --filter '@atproto/lex-cli...' build --force",
+    "precodegen": "pnpm run --recursive --stream --filter '@atproto/lex-cli...' build",
```

Comment:
I'm not sure how necessary this was - removing it saves about 10 seconds.

Reply:
It's useful when working on `lex-cli` to make sure the dependencies are re-built. I my experience, it sometimes happens that `tsc` will not re-build a file unless `--force` is added (though I've never been able to determine why, maybe git branch swaps ?).

It can be disabled here as long as devs working on `lex-cli` are aware that they may need to run `p run precodegen --force && p codegen` before they issue their last commit.

Reply:
I actually remember what causes this: if a dependency is changed, the dependents don't get rebuilt as the typescript compiler won't check for timestamp of files not in `src`.

This is only an issue if the changes in the dependencies influence the build of the codegen, which they usually shouldn't. So your change here should be fine.